Utilities/build-script-helper.py

```diff
@@ -50,25 +51,28 @@ def get_swiftpm_options(args):
       os.path.join(args.toolchain, 'lib', 'swift', 'Block'),
     ]
 
-    if 'ANDROID_DATA' in os.environ:
-      swiftpm_args += [
-        '-Xlinker', '-rpath', '-Xlinker', '$ORIGIN/../lib/swift/android',
-        # SwiftPM will otherwise try to compile against GNU strerror_r on
-        # Android and fail.
-        '-Xswiftc', '-Xcc', '-Xswiftc', '-U_GNU_SOURCE',
-      ]
-    else:
-      # Library rpath for swift, dispatch, Foundation, etc. when installing
-      swiftpm_args += [
-        '-Xlinker', '-rpath', '-Xlinker', '$ORIGIN/../lib/swift/linux',
-      ]
+  if 'ANDROID_DATA' in os.environ or (args.cross_compile_host and re.match(
+    'android-', args.cross_compile_host)):
+    swiftpm_args += [
+      '-Xlinker', '-rpath', '-Xlinker', '$ORIGIN/../lib/swift/android',
+      # SwiftPM will otherwise try to compile against GNU strerror_r on
+      # Android and fail.
+      '-Xswiftc', '-Xcc', '-Xswiftc', '-U_GNU_SOURCE',
+    ]
+  elif platform.system() == 'Linux':
```

Comment:
Was this check necessary for some reason now (previously it was just "else")? Mostly curious.

Reply:
Yes, I moved this entire block out of the non-Darwin block above, so this project could be cross-compiled from macOS too, then I had to make sure the linux rpath isn't added for Darwin.

Reply:
Note to self: don't review python code changes with whitespace changes ignored!